Scripts/GameManagement/LobbyManager.gd

```diff
@@ -1,6 +1,7 @@
 extends Node
 
-var Player = preload("res://Entities/Character/Player/Player.tscn")
+#var Player = preload("res://Entities/Character/Player/Player.tscn")
+var Player = preload("res://Entities/Character/Player/Classes/Soldier.tscn")
```

Comment:
We need this?

Reply:
I thought I deleted this in "Removed unnecessary comments" commit, it must have slipped by.